app/navigators/AppNavigator.tsx

```diff
@@ -51,28 +51,32 @@ const Stack = createNativeStackNavigator<AppStackParamList>()
 
 const AppStack = observer(function AppStack() {
   const { authUserStore } = useStores()
-  const { authorize, getCredentials, user } = useAuth0()
-
-  if (!user) {
-    console.tron.log("Auth0 Error calling getCredentials() - no user")
-  }
+  const { authorize, getCredentials, user, error } = useAuth0()
 
   if (user !== authUserStore.user) {
     authUserStore.saveUser(user)
   }
 
-  getCredentials().then(({ accessToken, idToken }) => {
-    if (accessToken !== authUserStore.access_token) {
-      authUserStore.saveAccessToken(accessToken)
-    }
+  if(authUserStore.user && !authUserStore.isLoggedIn){
```

Comment:
What happens if the user is signed in in the auth store but their token expires?